hugr-core/src/hugr/patch/simple_replace.rs

```diff
@@ -1146,156 +1040,48 @@ pub(in crate::hugr::patch) mod test {
         assert_eq!(h.entry_descendants().count(), 6);
     }
 
-    #[rstest]
```

Comment:
what's happened to these two tests

Reply:
They had been added in #2151 and covered the `OutputBoundaryMap` struct (now deleted), as well as the multiple ways in which the boundary map could be defined. I've now realised that we can get rid of all of that.